Swiftfin tvOS/Views/VideoPlayer/Overlays/Overlay.swift

```diff
@@ -69,6 +71,15 @@ extension VideoPlayer {
                     isPresentingOverlay = true
                     overlayTimer.start(5)
                 }
+                .onPlayPauseCommand(perform: {
+                    if isPlaying {
+                        proxy.pause()
+                        isPlaying = false
+                    } else {
+                        proxy.play()
+                        isPlaying = true
+                    }
```

Comment:
```suggestion
                    if !isPresentingOverlay {
                        isPresentingOverlay = true
                        overlayTimer.start(5)
                    }
                    switch videoPlayerManager.state {
                    case .playing:
                        proxy.pause()
                    default:
                        proxy.play()
                    }
```

Reply:
@LePips does this solve your change request?

Reply:
Not just seeing if the overlay is being presented but also set it to the main overlay.